src/Microsoft.ML.Data/Model/ModelOperationsCatalog.cs

```diff
@@ -26,19 +32,126 @@ internal ModelOperationsCatalog(IHostEnvironment env)
             Explainability = new ExplainabilityTransforms(this);
         }
 
+        private void Save<TSource>(DataViewSchema schema, IDataLoader<TSource> model, Stream stream)
```

Comment:
>ader<TSource> m [](start = 65, length = 15)

In this case where we have a data loader, we *have* the schema already, do we not?